edx_proctoring/templates/proctoring/seq_proctored_practice_exam_error.html

```diff
@@ -0,0 +1,56 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Your proctoring proctoring session is in error
+    {% endblocktrans %}
+  </h3>
+
+  <h4>
+    {% blocktrans %}
+      Your Practice Proctoring Session: <b class="failure"> Failed </b>
```

Comment:
@chrisndodge @griffresch @marcotuts Not sure what to recommend here. Seems a bit strange to talk about practice proctoring results in the same way as real proctoring review results. Should we keep this same pattern?
"Your practice proctoring results: Unsatisfactory"